features/events/api/src/main/java/org/opennms/netmgt/events/api/EventsWrapper.java

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2016 The OpenNMS Group, Inc.
```

Comment:
Make sure you update the copyrights in the files you create/change if possible, as well.